shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -885,6 +886,11 @@ export class AppInsightsCore<CfgType extends IConfiguration = IConfiguration> im
             function _initPluginChain(updateState: ITelemetryUpdateState | null) {
                 // Extension validation
                 let theExtensions = _validateExtensions(_self.logger, ChannelControllerPriority, _configExtensions);
+                let delayInit: IPlugin[] = [];
```

Comment:
Not exactly what I was thinking as the offline channel still needs to init before the post channel....

I was thinking that within the Offline channel it's dependencies on the Post Channel (Offline support) would be lazy so it doesn't try and "get" it until after initialization of itself.